cli/tests/error_015_dynamic_import_permissions.out

```diff
@@ -1 +1 @@
-error: Uncaught TypeError: network access to "http://localhost:4545/cli/tests/subdir/mod4.js", run again with the --allow-net flag
+error: network access to "http://localhost:4545/cli/tests/subdir/mod4.js", run again with the --allow-net flag
```

Comment:
`PermissionsDenied` should be shown here - without it this message is incomplete

Reply:
I am not sure what you are referring to... with or without this PR there is no `PermissionsDenied` text... if we want that, then it should be a subsequent PR.

Reply:
You're right, sorry for noise.